App/navigators/RootStack.tsx

```diff
@@ -37,6 +59,72 @@ const RootStack: React.FC = () => {
     navigation.navigate(Screens.Terms)
   }
 
+  const initAgent = async () => {
+    if (initAgentInProcess) {
+      return
+    }
+
+    Toast.show({
+      type: ToastType.Info,
+      text1: t('StatusMessages.InitAgent'),
+      position: 'bottom',
+    })
+
+    //Flag to protect the init process from being duplicated
+    setInitAgentInProcess(true)
+
+    //Remove before production
+    try {
+      const newAgent = new Agent(
+        {
+          label: 'Aries Bifold',
+          mediatorConnectionsInvite: Config.MEDIATOR_URL,
+          mediatorPickupStrategy: MediatorPickupStrategy.Implicit,
+          walletConfig: { id: 'wallet4', key: '123' },
```

Comment:
Based off of the PR description I expected to see the key being used here to be the user pin, or did I misunderstand something, or is that a followup/additional PR?

Reply:
There will be follow up PRs regarding that